src/config/constants/farmAuctions.ts

```diff
@@ -1,171 +1,172 @@
 import { Token as SDKToken, Pair, ChainId } from '@pancakeswap/sdk'
-import tokens from './tokens'
-import { FarmAuctionBidderConfig, Token } from './types'
+import { coreTokens, getToken } from './tokens'
+import { FarmAuctionBidderConfig } from './types'
 
-const getLpAddress = (token: string, quoteToken: Token) => {
+const getLpAddress = (token: string, quoteToken: SDKToken) => {
   const tokenAsToken = new SDKToken(ChainId.MAINNET, token, 18)
-  const quoteTokenAsToken = new SDKToken(ChainId.MAINNET, quoteToken.address[56], 18)
-  return Pair.getAddress(tokenAsToken, quoteTokenAsToken)
+  return Pair.getAddress(tokenAsToken, quoteToken)
 }
 
-export const whitelistedBidders: FarmAuctionBidderConfig[] = [
+const { WBNB, BUSD } = coreTokens
+
+const whitelistedBidders: FarmAuctionBidderConfig[] = [
```

Comment:
Using this file as an example of migration as it only uses tokens from the `coreTokens`